packages/spotify-web-api/spotify-web-api.0.2.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1"
+maintainer: "john.else@gmail.com"
+available: [ ocaml-version >= "4.01.0" ]
+build: [
+  [make]
+]
+build-test: [
+  [make "test"]
+]
+install: [
+  [make "PREFIX=%{prefix}%" "install"]
+]
```

Comment:
opam before 1.2.0 didn't use a separate `install` field in opam files (also build-test wasn't supported). If you put the install command in `build` (like with spotify-cli), this should work for older opams. @AltGr why doesn't opam 1.1 complain about not understanding this field?